### PR TITLE
Complete basic contributions.md content

### PR DESCRIPTION
**Pull request by kbeyls**

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -1,7 +1,97 @@
...
+  - We aim for individual commits adding new content to not be overly large, but
+    remain focussed on a specific topic.
+  - We aim for commits fixing style, spelling or other language issues to fix
+    one general issue at a time.
+  - When writing new content, do use the ``\todo`` macro to indicate if some
+    detail should or could be added later. Similar to how TODO or FIXME comments
+    are used in code projects, we use ``\todo`` macros to indicate an
+    improvement that should be made later. We hope that by using ``\todo``,
+    contributors will wrestle a bit less with writer's block. For now, to-dos
+    are only visible in the PDF output, where they are put in the margin. An example to-do looks as follows in the source:
```
Comment (kbeyls) [inline contributing.md:67]: An 80-column line break seems to be missing here.

Comment (kbeyls) [reply contributing.md]: Should now be fixed in the added commit e0cc963.

File: contributing.md [hunk 2]
```diff
@@ -1,7 +1,97 @@
...
+- Any other improvements.
+
+## What is the vision for the project?
+
+See https://github.com/llsoftsec/llsoftsecbook/blob/main/README.md#purpose
+
+## Submitting ideas for new content, how to improve language, ...
+
+Please raise an issue at https://github.com/llsoftsec/llsoftsecbook/issues/. It
+may be a good idea to check if there isn't already an issue for what you're
```
[the remark this answers is not in the document]
Comment (kbeyls) [reply contributing.md]: Should now be fixed in the added commit e0cc963.

File: contributing.md [hunk 3]
```diff
@@ -1,7 +1,103 @@
...
+    example to-do looks as follows in the source:
+    ``\todo{Also support \todo in the HTML output.}.``
+
+    For completely missing sections of contents, use
+    ``\missingcontent{A description of the missing content goes here.}``
+
+- For now, we keep all text content of the book in a single
+  [book.md](https://github.com/llsoftsec/llsoftsecbook/blob/main/book.md) file.
+- When adding new content, please remember to:
+  - Spell check your contributions. We use US English spelling.
```
Comment (fpetrogalli) [inline contributing.md:77]:
```suggestion
  - Spell check your contributions. Please check [General style and grammar conventions](general-style-and-grammar-conventions) for details.
```

**Pull request merged**